packages/sdk/src/swap/sdk.ts

```diff
@@ -33,9 +34,9 @@ import type {
 type TransactionHash = `0x${string}`;
 
 export type SwapSDKOptions = {
-  network?: ChainflipNetwork;
+  network: ChainflipNetwork;
+  backendUrl: string;
```

Comment:
i think this should be optional.

Reply:
Why are both not optional anymore?

Reply:
Ah, i see you declared the options in the constructor as `Partial`. Hmm. I guess that will work for the time being but once we add a new required property here, that will be set as Optional in the constructor type and someone might forget to change this. 
Can we somehow split the types of both `options` objects in a better way?

Reply:
added a `Required` helper to make properties of the class variable non nullable